content/blog/2018-05-02-onboarding-is-work.md

```diff
@@ -0,0 +1,415 @@
+review [an abnormally big
+package](https://ropensci.org/blog/2018/04/06/peer-review-value/) and
+some others not.
+
+### Work done by reviewers
+
+The previous figures of lines of code and NAMESPACE exports give a good
+idea of the size and complexity of the package authors have to review:
+Reviewers more or less get to review the package as it was at submission
+(minus editor remarks, and well the review of the first reviewer if the
+second one reviews much later). Now, we also ask reviewers to tell us
+how much time they spent reviewing, not to judge anyone, but to get a
+sense of the involvement we asked. We also use this information when
+recruiting reviewers, generally saying that reviewing a package is “as
+long as reviewing a scientific paper”, a couple of hours.
+
+``` r
+# airtable data
+# this is our private database of who's reviewed what
+# and of reviewers' areas of expertise
+airtable <- airtabler::airtable("appZIB8hgtvjoV99D", "Reviews")
+airtable <- airtable$Reviews$select_all()
+```
+
+Out of 184 we now have reviewing time for 136.
+
+``` r
+ggplot(airtable) +
+  geom_boxplot(aes(y = review_hours, x = "")) +
+  hrbrthemes::theme_ipsum(base_size = 16,
+                          axis_title_size = 16) +
+  theme(axis.title.x=element_blank()) +
+  ylab("Reviewing time (hours)")
+```
+
+![hours spent
+reviewing](/img/blog-images/2018-05-02-onboarding-is-work/unnamed-chunk-8-1.png)
+
+In case you wonder, what’s in it for reviewers? They do it out of
+personal interest for scientific software, to learn more about package
+development, to discover a package, and maybe because they feel grateful
+for reviews of their own software. You can read personal perspectives of
+reviewers on our blog: [one by Mara
+Averick](https://ropensci.org/blog/2017/08/22/first-package-review/),
+[one by Verena
+Haunschmid](https://ropensci.org/blog/2017/09/08/first-review-experiences/),
+[one by Charles T.
+Gray](https://ropensci.org/blog/2018/03/13/ode-to-testing/) and [one by
+Miles McBain](https://ropensci.org/blog/2018/04/06/peer-review-value/).
+Some reviewers end up being listed as reviewers in the package list of
+authors as explained [in this insightful and entertaining post by editor
+Noam
+Ross](https://ropensci.org/blog/2018/03/16/thanking-reviewers-in-metadata/).
+
+#### Is the size of the package related to its review time
+
+``` r
+namespace_ro <- dplyr::left_join(namespace_ro, airtable, by = "package")
+ggplot(namespace_ro) +
+  geom_point(aes(exports, review_hours))
+```
+
+![](/img/blog-images/2018-05-02-onboarding-is-work/scatterplot-size-vs-reviewing-time-1.png)
+
+I’d have been surprised to see a link here anyway!
+
+#### Work done by editors
+
+Editors perform a few checks before seeking reviewers, and then
+generally supervise the process. In [Tim Trice’s
+words](https://ropensci.org/blog/2017/09/27/rrricanes/), we are “guiding
+angels from start to finish during the entire onboarding and review
+process”.
+
```

Comment:
I find it a little over the top to self-compliment ourselves so hard in our own post.